core/src/test/scala/org/apache/spark/ui/jobs/AllJobsPageSuite.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ui.jobs
+
+import javax.servlet.http.HttpServletRequest
+
+import org.mockito.Mockito._
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.status.AppStatusStore
+import org.apache.spark.status.api.v1
+
+class AllJobsPageSuite extends SparkFunSuite {
```

Comment:
Is it sufficient to add a test to `UISeleniumSuite` rather than adding a new Suite?

Reply:
Hi @sarutak I found that there are tests for method `render`,So we can remove `getSchedulingMode`,please review, Thanks very much!

Reply:
The latest change doesn't include test. Could you add one to `UISeleniumSuite`?

Reply:
> The latest change doesn't include test. Could you add one to `UISeleniumSuite`?

oh, Because i found that there are tests for method `render` yestoday.

Reply:
I mean we need to test whether the page is rendered and the scheduling mode is displayed with no error when we visit `http://localhost:4040/jobs/` even though we configure the scheduling mode like `spark.scheduler.mode=fair` instead of `spark.scheculer.mode=FAIR`.
Currently, we can get the following error when we visit `http://localhost:4040/jobs/` if we set the scheduling mode in lower case.
![scheduling_mode_in_lower_case](https://user-images.githubusercontent.com/4736016/103968628-87859900-51a7-11eb-994f-eb07a158c526.png)

Reply:
In `UISeleniumSuite.scala`，due to import scalaTest，I may not be able to mock，and I see that `AllJobsPage.scala`  doesn't has own suite，May i add a new suite？thx

Reply:
Hi @sarutak ，I've added the test in `UISeleniumSuite`, thx.

Reply:
What I intend is adding a test using Selenium like other tests in `UISeleniumSuite`., then you don't need to add `getScheduleMode`.
Here is an example.
https://github.com/apache/spark/blob/71d261ab8fb3e7fb22d2687b8e038129ca766a65/core/src/test/scala/org/apache/spark/ui/UISeleniumSuite.scala#L723-738

Reply:
Hi @sarutak , I don't understand that can i get <strong> element?
`val conf = Map("spark.scheduler.mode" -> "fair")
    withSpark(newSparkContext(additionalConfs = conf)) { sc =>
      val rdd = sc.parallelize(0 to 100, 100).repartition(10).cache()
      rdd.count()

      eventually(timeout(5.seconds), interval(50.milliseconds)) {
        goToUi(sc, "/jobs")
        find(linkText("Scheduling Mode")).get.text should contain("FAIR")
      }
    }`

Reply:
Hi @sarutak , I am so sorry，I haven't created PR before，I understand what you mean now。

I have removed `getSchedulingMode` and add a suit like other tests in `UISeleniumSuite`,Thanks very much!